src/config/db_config.js

```diff
@@ -0,0 +1,4 @@
+const config = require('../database/knexfile')[process.env.NODE_ENV]
```

Comment:
I would merge `db_config.js` & `knexfile.js` into a single file `db_config.js`. Having 2 files here isn't really giving you any benefit. Also both these files are related to the db config, which can be combined